src/plugins/node/loop/nodeloopplugin.cpp

```diff
@@ -68,13 +68,16 @@ static void GetLoopInformation(uv_timer_s *data) {
 #else
 static void GetLoopInformation(uv_timer_s *data, int status) {
 #endif
+    double mean = std::numeric_limits<double>::infinity();
+    if(num != 0)
+        mean = sum / num;
```

Comment:
I made that change because when starting and stopping the plugin, there may be no accumulated data. So `num` might be 0 at the time this event is fired, which then causes a divide by 0. This is to protect against that corner case. 

I made that change because when starting and stopping the plugin there might be a time when the accumulated data so `num` would be 0. This is to protect against that.